tests/behat/features/bootstrap/RaContext.php

```diff
@@ -102,6 +102,18 @@ public function iAmLoggedInIntoTheRaPortalAs($userName, $tokenType)
         $this->minkContext->assertPageContainsText('Token activation');
     }
 
+    /**
+     * @Given /^I am on RA "([^"]*)"$/
```

Comment:
This one bothers me some. It is not very clear what RA means in this context. I suggest:

> I visit the "([^"]*)" page

Or

> I visit the "([^"]*)" page in the RA environment

This is more in line with the way the other navigation givens work.

Reply:
A good suggestion!